packages/flash-swap/test/integration/hifiFlashUniswapV2/effects/uniswapV2Call.ts

```diff
@@ -227,8 +227,6 @@ export default function shouldBehaveLikeUniswapV2Call(): void {
                 const localToken1Amount: BigNumber = WBTC("0");
 
                 beforeEach(async function () {
-                  await this.contracts.uniswapV2Pair.__godMode_setToken0(this.contracts.usdc.address);
```

Comment:
Removing this makes nullifies the test. As per the context defined a few lines above:

> when the token0 is the underlying